### PR TITLE
Bugfix/fixing attrs

### PR DESCRIPTION
**Pull request by paganotoni**
This uses downcase for GoType and Adds fizzCol function for commonType

### CODE REVIEW

**Comment by paganotoni** [comment]
@markbates this one is required to get buffalo-auth columns all-set

File: movinglater/attrs/attrs.go [hunk 1]
```diff
@@ -45,10 +49,37 @@ func (a Attr) GoType() string {
...
+	case "nulls.float32", "nulls.float64":
+		return "float"
+	case "slices.string", "slices.uuid", "[]string":
+		return "varchar[]"
+	case "slices.float", "[]float", "[]float32", "[]float64":
+		return "numeric[]"
+	case "slices.int":
+		return "int[]"
+	case "slices.map":
+		return "jsonb"
```
Comment (markbates) [inline movinglater/attrs/attrs.go]: json

File: movinglater/attrs/attrs.go [hunk 2]
```diff
@@ -45,10 +49,37 @@ func (a Attr) GoType() string {
...
+	case "int":
+		return "integer"
+	case "time", "datetime":
+		return "timestamp"
+	case "uuid.uuid", "uuid":
+		return "uuid"
+	case "nulls.float32", "nulls.float64":
+		return "float"
+	case "slices.string", "slices.uuid", "[]string":
+		return "varchar[]"
```
Comment (markbates) [inline movinglater/attrs/attrs.go]: []string

File: movinglater/attrs/attrs.go [hunk 3]
```diff
@@ -45,10 +49,37 @@ func (a Attr) GoType() string {
 }
 
 func (a Attr) CommonType() string {
-	if a.commonType != "" {
-		return a.commonType
+	return fizzColType(a.commonType)
+}
+
+func fizzColType(s string) string {
```
Comment (markbates) [inline movinglater/attrs/attrs.go]: Remove fizz from this. We’re trying to keep this general. :)

File: movinglater/attrs/attrs.go [hunk 4]
```diff
@@ -45,10 +49,37 @@ func (a Attr) GoType() string {
...
+	case "time", "datetime":
+		return "timestamp"
+	case "uuid.uuid", "uuid":
+		return "uuid"
+	case "nulls.float32", "nulls.float64":
+		return "float"
+	case "slices.string", "slices.uuid", "[]string":
+		return "varchar[]"
+	case "slices.float", "[]float", "[]float32", "[]float64":
+		return "numeric[]"
```
Comment (markbates) [inline movinglater/attrs/attrs.go]: []float

File: movinglater/attrs/attrs.go [hunk 5]
```diff
@@ -45,10 +49,37 @@ func (a Attr) GoType() string {
...
+	case "slices.int":
+		return "int[]"
```
Comment (markbates) [inline movinglater/attrs/attrs.go]: []int

**Comment by paganotoni** [comment]
Seems like it doesn't, 
p.e: https://github.com/gobuffalo/fizz/blob/master/translators/postgres.go#L201

**Comment by markbates** [comment]
they're not. once the attrs stuff is finished then we can tackle moving migration generators there from soda

**Comment by paganotoni** [comment]
Oh got it, so this PR will not fix our auth plugin 😕, that will require some other PR's

**Comment by paganotoni** [comment]
I think we're all set here with the changes, LMK when you have the chance to review it, thanks Mark!

**Pull request merged**